src/System.CodeDom/src/System/CodeDom/Compiler/TempFiles.cs

```diff
@@ -112,7 +112,6 @@ private void EnsureTempNameCreated()
                     _basePath = Path.Combine(
                         string.IsNullOrEmpty(TempDir) ? Path.GetTempPath() : TempDir,
                         Path.GetFileNameWithoutExtension(Path.GetRandomFileName()));
-                    string full = Path.GetFullPath(_basePath);
```

Comment:
Good spot on the dead code.  This had been part of a permission demand, which was removed, and I neglected to remove the code that was providing the input to the demand.